hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientAbstract.java

```diff
@@ -1029,6 +1039,33 @@ private void bucketUsedBytesTestHelper(BucketLayout bucketLayout)
         store.getVolume(volumeName).getBucket(bucketName).getUsedBytes());
   }
 
+  private void bucketUsedBytesOverWrite(BucketLayout bucketLayout)
```

Comment:
Test looks good, but can we use `org.junit.jupiter.params.ParameterizedTest` for the bucket layout types, instead of new test methods, similar to what the EC tests are doing for replication config in this class?

Reply:
Sure. Let's change test using ParameterizedTest.